scripts/print-example-ci-command.js

```diff
@@ -4,7 +4,7 @@ const fg = require('fast-glob');
 
 const packageJSON = fg.sync(['examples/**/package.json'], { ignore: ['**/node_modules/**'] });
 
-const ignoredPackages = ['example-react-nextjs-swr'];
+const ignoredPackages = [];
```

Comment:
We can now remove the example from the ignored array 🎉 

I'll keep the current implementation in case we need to ignore tests in the future (hope not! 🤞 )